testing/README.md

```diff
@@ -0,0 +1,76 @@
+### E2E tests
+
+#### Dependencies
+
+[oblt-cli](https://studious-disco-k66oojq.pages.github.io)
+
+### Try it locally
+
+0. Setup vault
+`oblt-cli` stores cluster secrets in vault. 
+Export the environment variable `VAULT_TOKEN` or have the `~/.vault-token` file created.
+For further details about how to configure Vault check [the Vault documentation](https://github.com/elastic/infra/tree/master/docs/vault)
+
+1. Install and configure [oblt-cli](https://studious-disco-k66oojq.pages.github.io)
+
+2. Create a cluster for e2e tests:
+
+In the `testing` directory:
+
+```
+GITHUB_TOKEN=... SLACK_CHANNEL=... make create-cluster
+```
+
+Specify a `GITHUB_TOKEN` with configured SSO and the `SLACK_CHANNEL` - the slack channel identifier to send messages about cluster state. 
+
+It uses [oblt-cli](https://studious-disco-k66oojq.pages.github.io) to spin up an ephemeral cluster with ES, Kibana, APM and fleet.
+
+The cluster specs are located in `test-cluster.yml.tpl`. 
+
+The `create-cluster` target creates 2 files: 
+* `cluster-info.json` - Basic cluster information
+* `cluster-digest.yml` - a sensitive YAML containing all required endpoints and credentials to be passed to tests in the next step.
```

Comment:
I assume these files should be created in the same folder where `make create-cluster` was run from.  I'm not seeing them there.  I'm guessing that's because of the error I mentioned in https://github.com/elastic/elastic-agent/pull/2292/files#r1159970974, so I'm going to stop here for the moment, while https://github.com/elastic/elastic-agent/pull/2292/files#r1159970974 is resolved.